ports/pcre/portfile.cmake

```
@@ -20,7 +20,7 @@ vcpkg_from_sourceforge(
 )
 
 set(IS_PCRE_SUPPORT_JIT YES)
-if(VCPKG_TARGET_ARCHITECTURE MATCHES "loongarch")
+if(VCPKG_TARGET_ARCHITECTURE MATCHES "(loongarch|^s390|^arm)")
```

I don't see failures for arm android/osx/windows CI.
```suggestion
if(VCPKG_TARGET_ARCHITECTURE MATCHES "(loongarch|^s390)")
```

---

(Maybe CI doesn't test jit. IDK.)

---

Ah, I succeded to build it on armv7l but missed it before.